### ISSUE TITLE
Dead instances/Removal of communities

### ISSUE THREAD

**Issue by chris-y** [fnic-lemmy/lemmy-directory#63]
kbin.social has been dead for a while.  Should we remove communities hosted on it?
feddit.de is down - AFAIK the owner abandoned it, should we remove feddit.de communities?

feddit.de has largely been replaced by feddit.org and I know some of the kbin.social communities have been moved (HistoryRuins etc)

**Comment by M4rtineski** [comment]
Yes, I planned to do a cleanup for dead communities because they're not coming back. But it would still be good to track the instances that died for easier crosschecking to see if the community is just temporarily down or if it's one of those instances.

**Comment by M4rtineski** [comment]
Cleaned up.
